pybossa/syncer/project_syncer.py

```diff
@@ -81,6 +81,13 @@ def _build_payload(self, project, target=None):
 
         if not target:
             payload = self._remove_reserved_keys(project_dict)
+            input_data = project['info'].get('data_classification', {}).get('input_data') or ''
+            output_data = project['info'].get('data_classification', {}).get('output_data') or ''
+            data_class = str(input_data) + str(output_data)
+            # using .get in case annotation_config doesn't exist 
+            annotation_config = payload['info'].get('annotation_config', {})
+        if ('L1' in data_class or 'L2' in data_class):
+            annotation_config['amp_store'] = False
```

Comment:
`annotation_config` is declared the `if not target` but used in`if ('L1' in data_class or 'L2' in data_class):` There are chances that annotation_config is referenced without declared.

Reply:
the travis build failed for similar reason: https://travis-ci.com/github/bloomberg/pybossa/builds/225877069